src/components/search/Search.js

```diff
@@ -1,14 +1,81 @@
-import { SearchAlt2 } from '@styled-icons/boxicons-regular'
+import '@reach/combobox/styles.css'
+
+import {
+  Combobox,
+  ComboboxInput,
+  ComboboxList,
+  ComboboxOption,
+} from '@reach/combobox'
+import { useHistory } from 'react-router-dom'
+import usePlacesAutocomplete, {
+  getGeocode,
+  getLatLng,
+} from 'use-places-autocomplete'
+import { NumberParam, useQueryParams } from 'use-query-params'
 
 import Input from '../ui/Input'
+import SearchEntry from './SearchEntry'
+
+const Search = () => {
+  const [centerCoords, setCenterCoords] = useQueryParams({
+    centerLat: NumberParam,
+    centerLng: NumberParam,
+  })
+
+  let history = useHistory()
```

Comment:
Can we change this to a `const` since we're only mutating `history`?

Reply:
+1